### PR TITLE
Page is publishable second step

### PR DESCRIPTION
**Pull request by md8n**
# Description

This adds functionality to get Assets that are referred to by a Page. 
And then to check on the status of those Assets in determining whether a Page `isPublishable`.

To support the custom strategy upgrading workbox to v6+ was required.  That in turn required upgrading yarn.

### CODE REVIEW

File: src/ts/StartUp.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export const initialiseCertChain = async (): Promise<void> => {
             }
         } catch (e) {
             // No signed cert - change this to a proper message, if that is appropriate
-            console.info(e);
+            // console.info(e);
```
Comment (PeteCoward) [inline src/ts/StartUp.ts:51]: YAY!

File: src/ts/Constants.ts [hunk 2]
```diff
@@ -1,5 +1,12 @@
 /* A place to define various magic strings etc. that do not belong in urls.js */
 
+// Cache names
 export const MANIFEST_CACHE_NAME = "manifest-cache";
 export const PAGES_CACHE_NAME = "pages-cache";
```
Comment (PeteCoward) [inline src/ts/Constants.ts:5]: can we delete this?

Comment (md8n) [reply src/ts/Constants.ts:5]: yep

Comment (md8n) [reply src/ts/Constants.ts:5]: Still there. Let's get rid of it when we actually clean out sitedownloader and cachedisplay

File: src/ts/Implementations/Manifest.ts [hunk 3]
```diff
@@ -66,91 +83,88 @@ export class Manifest implements TManifest {
...
+            this.status = "prepped";
+        }
+    }
+
+    get updatedResp(): Response {
+        return new Response(JSON.stringify(this.data));
+    }
+
+    get cacheKey(): string {
+        return this.fullUrl;
```
Comment (PeteCoward) [inline src/ts/Implementations/Manifest.ts:131]: should not be the fullUrl, but the storagecontainer name returned in the page entry in the manifest

**Review by PeteCoward — APPROVED** [review]

**Pull request merged**